docs/reference/migration/migrate_6_8.asciidoc

```diff
@@ -0,0 +1,10 @@
+[[breaking-changes-6.8]]
+== Breaking changes in 6.8
+++++
+<titleabbrev>6.8</titleabbrev>
+++++
+
+There are no breaking changes for 6.8.0
```

Comment:
@jasontedor - Is there anything a user needs to know in reference to https://github.com/elastic/elasticsearch/pull/41701 ? (or any other changes with this release ?)

Reply:
Nope, we should be good here as that is not a breaking change (it does change the keystore format, but we handle it automatically on upgrades). It has the consequence that a user can not upgrade from 6.8.0 to 7.0.0/7.0.1 but that would fall outside of guarantees (latest to latest) and be unusual anyway (since 7.1.0 would be the logical choice).